demo/screens/HomeScreen.js

```diff
@@ -1,8 +1,15 @@
 import React from "react";
 import Home from "@presentational/Home";
 import { NavigationEvents } from "react-navigation";
+import { Action } from "tasit-sdk";
+import tasitSdkConfig from "../config/default.js";
 
 export default class HomeScreen extends React.Component {
+  componentDidMount() {
```

Comment:
Let's make sure this screen shows something nice-looking if the connection to the blockchain can't be estabilished.

Reply:
@marcelomorgado But we can just keep this in mind for the future. At most with a comment in the code.

Reply:
OK, should I create an issue for that too?

Reply:
Sure. Thanks. Resolving this.

Reply:
Or wait, I shouldn't resolve it until we create the issue. But you can resolve this.

Reply:
:+1:

Reply:
https://github.com/tasitlabs/tasit/issues/83. Resolving this.